shared/common-adapters/popup-menu.desktop.js

```diff
@@ -141,7 +141,6 @@ const stylesMenu = {
 }
 
 const stylesMenuText = {
-  ...globalStyles.clickable,
```

Comment:
Have to remove this, otherwise disabled menu items still get the hand cursor

Reply:
do the other items still have it?

Reply:
Yeap. stylesMenuText is used only for the children of the Box for the menu item, and Box has `styleClickable`, which switches on `disabled` correctly.